Extensions/3D/Model3DRuntimeObject3DRenderer.ts

```diff
@@ -113,6 +113,66 @@ namespace gdjs {
 
       return aabb;
     }
+
+    _updateMaterials() {
+      // @ts-ignore It can't be null if THREE exists.
+      const originalModelMesh: THREE.Object3D = this._model3DRuntimeObject
+        .getInstanceContainer()
+        .getGame()
+        .getModel3DManager()
+        .getModel(this._model3DRuntimeObject._modelResourceName);
+      const modelObject3D = originalModelMesh.clone();
+
+      this.get3DRendererObject().remove(this._threeObject);
+      this.get3DRendererObject().add(modelObject3D);
+
+      this._threeObject = modelObject3D;
+
+      this._replaceMaterials();
+    }
+
+    /**
+     * Replace materials to better work with lights (or no light).
+     */
+    _replaceMaterials() {
+      if (
+        this._model3DRuntimeObject._materialType ===
+        gdjs.Model3DRuntimeObject.MaterialType.StandardWithoutMetalness
+      ) {
+        this._threeObject.traverse((node) => {
+          if (node.type === 'Mesh') {
+            const mesh = node as THREE.Mesh;
+            const material = mesh.material as THREE.MeshStandardMaterial;
+            //@ts-ignore
+            if (material.metalness) {
+              //@ts-ignore
+              material.metalness = 0;
+            }
+          }
+        });
+      } else if (
```

Comment:
I see no action based on material type being "Keep original". Does this mean that a model that is configured KeepOriginal won't be updated if its setting is changed and then hot-reloaded?

Reply:
It starts from the original model every time (cf the `clone()`).

Reply:
👍